CONTRIBUTING.md

```diff
@@ -108,6 +108,25 @@ to make that happen.
 
 Happy developing!
 
+### Using DummyAuthenticator & SimpleSpawner
+
+The default JupyterHub [authenticator](https://jupyterhub.readthedocs.io/en/stable/reference/authenticators.html#the-default-pam-authenticator)
```

Comment:
To simplify testing of JupyterHub, it's helpful to use DummyAuthenticator instead of the default JupyterHub authenticator and SimpleSpawner instead of the default spawner.